docs/user/masternodes/understanding.rst

```diff
@@ -81,6 +84,24 @@ real-time payment rates, and `this site
 <http://178.254.23.111/~pub/Dash/Dash_Info.html>`_ for various real-time
 statistics on the masternode network.
 
+High-Performance Masternodes (HPMNs)
+------------------------------------
+
+High-Performance Masternodes (HPMNs) are a subset of Masternodes that have 
+been created to host the Dash Platform. An HPMN is a lot like Masternodes 
+with these differences:
+
++----------------+-----------------------------------+-----------------------------------------------------+
+|                | Masternode                        | HPMN                                                |
++================+===================================+=====================================================+
+| Collateral     | 4 times less collateral than HPMN | Has four times the collateral as normal masternodes |
++----------------+-----------------------------------+-----------------------------------------------------+
+| Specs          | Lesser than HPMN                  | Higher than normal masternodes                      |
++----------------+-----------------------------------+-----------------------------------------------------+
+| Service        | Only Dash Core                    | Both Dash Core and Platform                         |
++----------------+-----------------------------------+-----------------------------------------------------+
+| Voting Weight  | 4 times less than HPMN            | 4 times more than HPMN                              |
++----------------+-----------------------------------+-----------------------------------------------------+
```

Comment:
I think this table would make more sense by having the left column specify actual values (1000 Dash, 1 node 1 vote) and the right column specify the difference `4x masternode collateral (4000 Dash)`, rather than have each column compare against the other. The current description implies that any value differing by a factor of 4 would be sufficient, e.g. 200 & 800 Dash collaterals.

Reply:
Hey @strophy we decided to not specify the collateral (1000 DASH) since it could change. That's why it has been worded like that. I will change the voting thing though.

 CC @thephez